data/json/items/gun/shot.json

```diff
@@ -211,7 +236,11 @@
     "durability": 7,
     "barrel_length": 3,
     "magazine_well": 1,
-    "magazines": [ [ "shot", [ "saiga10mag", "saiga30mag" ] ] ],
+    "magazines": [
+      [ "shot", [ "saiga10mag", "saiga30mag" ] ],
+      [ "308", [ "falmag", "falbigmag" ] ],
```

Comment:
Saiga semi-automatic rifle variants which use .223 and .308 ammo exist. The only thing I'm not sure about is which magazine should be used for .308 conversion.

https://en.wikipedia.org/wiki/Saiga_semi-automatic_rifle